coordinated-entry-screening/static/css/custom.css

```diff
@@ -1,6 +1,66 @@
+:root {
+    --blue: #0178BC;
+    --light-blue: #4da0d0;
+}
+
+/*http://getbootstrap.com/docs/4.0/examples/sticky-footer/*/
+html {
+  position: relative; 
+  min-height: 100%;
+}
+
+body {
+    font-family: 'Libre Franklin', sans-serif !important;
+    margin-bottom: 60px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: 800;
+}
+
+/* Custom Bootstrap utilities */
+.mt-6 {
+    margin-top: 4em !important; /* https://getbootstrap.com/docs/4.1/utilities/spacing/#examples */
+}
+
+.badge-primary {
+    background-color: var(--blue);
+}
+
+.btn-blue {
+    background-color: var(--blue);
+    color: #fff;
+}
+
+.btn-blue:hover, .btn-blue.active {
+    background-color: var(--light-blue);
+}
+
+/* Navbar and footer */
+nav {
+    background-color: var(--blue);
+    color: #fff;
+}
+.navbar-dark .navbar-nav .nav-link {
+    color: #fff;
+}
+
+.footer {
+    background-color: var(--light-blue);
+    color: #fff; 
+    line-height: 60px;
+      position: absolute;
+    bottom: 0;
+    width: 100%;
+}
+
+/* Dashboard charts */
 .highcharts-tooltip span {
     height: auto;
     width: 200px;;
     overflow: auto;
     white-space: normal !important;
```

Comment:
Let's modify this in [HighCharts](https://api.highcharts.com/highcharts/tooltip.style.whiteSpace) versus our CSS.

Reply:
Can you say a little more about "why"? My inclination is to put all the tooltips styles in the CSS, so as to avoid code duplication in the JS (since we have two chart helper functions).

Reply:
In general, it's bad practice to use `!important` and it should be avoided if possible. Right now your `white-space` is being overridden by that `whiteSpace` attribute in HighCharts, which is why you need the `!important`. Changing it in HighCharts eliminates that line; however I do see your point about redundancy.

Reply:
Defining the whiteSpace attribute as a Highchart property does not have the same affect. Instead, the text spills outside the box. Are you able to make it work?

```
tooltip: {
            useHTML: true,
            headerFormat: '',
            pointFormat:  '<h6><strong>{point.y} recommendations</strong></h6><p>{point.name}</p>',
            shadow: false,
            borderColor: '#3B4B5C',
            style: {
                whiteSpace: 'normal',
            }
        },
```

Reply:
Hm, I can't get this to work either...that seems like a HighCharts issue...

I don't think there should be a comma after `'normal'` but even fixing that doesn't work. Let's leave it with the `!important` so you can deploy. It's not a large issue.